pkg/test/istio.io/README.md

```diff
@@ -0,0 +1,10 @@
+# Istio.io Example Tests
+
+## Overview
+This folder contains a test framework as well as tests for the different examples
```

Comment:
I don't think you need to mention the test frame work here.  I think it's sufficient to just talk about how this is used by istio.io, etc.